app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ Required ] I think it would be better if your design for the registration page matches the design on the project wireframe. Currently, your design differs from the project wireframe in the following areas.
   
   - [ ]  The registration form should be centered in the middle of the page, currently, the form in your design is not centered.
   - [ ]  The `SignUp` button and the `input` elements should have the same width just the way it is in the project wireframe. Please see the screenshot below for more details on this.


## Your registration form design
![Screenshot from 2023-07-07 01-40-45](https://github.com/muhammedb8826/blog-app/assets/16392046/aa5442ba-0402-40b6-866b-aa7eefd6d27e)

## Project wireframe design
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)